electricitymap/contrib/lib/models/events.py

```diff
@@ -655,20 +655,19 @@ def _update(
             raise ValueError(
                 f"Cannot update events from different datetimes: {event.datetime} and {new_event.datetime}"
             )
-        if event.source != new_event.source:
-            raise ValueError(
-                f"Cannot update events from different sources: {event.source} and {new_event.source}"
-            )
         if event.sourceType != new_event.sourceType:
             raise ValueError(
                 f"Cannot update events from different source types: {event.sourceType} and {new_event.sourceType}"
             )
         production_mix = ProductionMix._update(event.production, new_event.production)
         storage_mix = StorageMix._update(event.storage, new_event.storage)
+        source = ", ".join(
+            set(event.source.split(", ")) | set(new_event.source.split(", "))
```

Comment:
This ensures that if we where to update and already updated list that now contain two sources these are deduplicated.